### PR TITLE
Re-factored code, removed code dublication, some formatting changes

### PR DESCRIPTION
**Pull request by dmitrijev**

### CODE REVIEW

File: classes/XLite/Module/Gingerpayments/Merchant/Main.php [hunk 1]
```diff
@@ -152,11 +152,10 @@ public static function debugLog($message = null, $data = null)
      */
     public static function includeGingerLib()
     {
-        $gingerPaymentsLibPath = LC_DIR_MODULES.'Gingerpayments'
```
Comment (mgsander) [inline classes/XLite/Module/Gingerpayments/Merchant/Main.php]: There was no `LC_DS` between `LC_DIR_MODULES` and `'Gingerpayment'` before, but in the updated version there is. Is this a problem?

File: classes/XLite/Module/Gingerpayments/Merchant/Model/Payment/Processor/Ginger.php [hunk 2]
```diff
@@ -306,8 +301,30 @@ protected function getGingerSettings()
...
+            'amount' => \XLite\Module\Gingerpayments\Merchant\Main::getAmountInCents(
+                $this->transaction->getOrder()->getTotal()
+            ),
+            'currency' => $this->getCurrency(),
+            'merchant_order_id' => $this->getTransactionId(),
+            'return_url' => $this->getReturnURL(false, true),
+            'description' => $this->getInvoiceDescription(),
+            'customer' => $this->getCustomerInformation(),
+            'payment_details' => array(),
+            'issuer_id' => \XLite\Core\Request::getInstance()->issuer ?: null,
```
Comment (mgsander) [inline classes/XLite/Module/Gingerpayments/Merchant/Model/Payment/Processor/Ginger.php]: Are empty `issuer_id` properties filtered out in the request?

File: classes/XLite/Module/Gingerpayments/Merchant/Main.php (hunk 1 above)
Comment (dmitrijev) [inline classes/XLite/Module/Gingerpayments/Merchant/Main.php]: Should not be a problem, but I can move `LC_DIR_MODULES` out of the `implode()` function

File: classes/XLite/Module/Gingerpayments/Merchant/Model/Payment/Processor/Ginger.php (hunk 2 above)
Comment (dmitrijev) [inline classes/XLite/Module/Gingerpayments/Merchant/Model/Payment/Processor/Ginger.php]: Yes: https://github.com/gingerpayments/ginger-php/blob/master/src/Client.php#L367

**Pull request merged**